pyproject.toml

```diff
@@ -13,11 +13,17 @@ include = ["LICENSE"]
 
 [tool.poetry.dependencies]
 python = "^3.9"
-pydantic = "^1.9.1"
+pydantic = "^2.7.0"
+pydantic-core = "^2.16.3"
+pytest = "^8.1.1"
+pluggy = "^1.4.0"
+bump-pydantic = "^0.8.0"
```

Comment:
`bump-pydantic` is helper package for your local usage. You don't need it as dependency